types/index.d.ts

```diff
@@ -0,0 +1,27 @@
+export default pluginCreator;
+/**
+ * @param {{precision?: number | false,
+ *          preserve?: boolean,
+ *          warnWhenCannotResolve?: boolean,
+ *          mediaQueries?: boolean,
+ *          selectors?: boolean}} opts
+ */
+declare function pluginCreator(opts: {
+    precision?: number | false;
+    preserve?: boolean;
+    warnWhenCannotResolve?: boolean;
+    mediaQueries?: boolean;
+    selectors?: boolean;
+}): {
```

Comment:
Will be great to use `@typdef` for options, so other developers can import them and use, for example `const options: postcssCalcOptions = { ...options };`

Reply:
Done.